samcli/lib/deploy/deployer.py

```diff
@@ -457,14 +460,16 @@ def wait_for_execute(self, stack_name: str, stack_operation: str, disable_rollba
             The type of the stack operation, 'CREATE' or 'UPDATE'
         disable_rollback : bool
             Preserves the state of previously provisioned resources when an operation fails
+        execution_time : float
+            Changeset execution time (current time)
```

Comment:
Can we update the description to explain clearer about what this argument is for?

Reply:
I've improved the documentation